src/test/datascience/notebook/remoteNotebookEditor.vscode.common.ts

```diff
@@ -209,25 +211,5 @@ export function sharedRemoteNotebookEditorTests(
         await Promise.all([runCell(cell), waitForTextOutput(cell, '123412341234')]);
     });
 
```

Comment:
So the kernel doesn't get auto selected here, so I removed the test. With only a plain metadata file and no preferredInterpreter this expected.

Also I don't believe this test was correct in any case. It's checking "preferred" in the end assert. But that value is a `void` that only has to do with starting up the jupyter server, it's not related to the preferred kernel.